serialization-avro/src/main/java/io/axoniq/dev/samples/serializationavro/command/ExceptionWrappingHandlerInterceptor.java

```diff
@@ -0,0 +1,54 @@
+package io.axoniq.dev.samples.serializationavro.command;
+
+import io.axoniq.dev.samples.serializationavro.api.GiftCardBusinessError;
+import io.axoniq.dev.samples.serializationavro.api.GiftCardBusinessErrorCode;
+import org.axonframework.commandhandling.CommandExecutionException;
+import org.axonframework.commandhandling.CommandMessage;
+import org.axonframework.messaging.InterceptorChain;
+import org.axonframework.messaging.MessageHandlerInterceptor;
+import org.axonframework.messaging.unitofwork.UnitOfWork;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.annotation.Nonnull;
+import java.lang.invoke.MethodHandles;
+
+public class ExceptionWrappingHandlerInterceptor implements MessageHandlerInterceptor<CommandMessage<?>> {
```

Comment:
I don't think we necessarily need the distributed exception handling sample inside the Avro Serializer sample. 
At least, I am guessing this was part of the copy/paste to get a usable application?

Reply:
Yes. You got me, I just copied over it from the example that you have, we can simplify it by removing the interceptor / error code mapping and just throw exceptions instead... Speed of implementation was my major concern, so I left it in. But we can easily drop it ...